taskcluster/translations_taskgraph/transforms/skip_unless_pipeline_changed.py

```diff
@@ -36,9 +36,17 @@
 transforms = TransformSequence()
 
 
+def check(files_changed, patterns):
```

Comment:
Is this check something re-usable?

Reply:
I guess ideally we would implement this as an optimization, since that's where this code moved: https://github.com/taskcluster/taskgraph/blob/eefda885ea20bb146e703282eb6e80247e651472/src/taskgraph/optimize/strategies.py#L70

Let me give that a go; it's a much better solution!